src/globus_cli/exception_handling/hooks.py

```diff
@@ -56,18 +63,8 @@ def _jsonpath_from_pydantic_loc(loc: list[str | int]) -> str:
     return path
 
 
-_DEFAULT_SESSION_REAUTH_MESSAGE = (
-    "The resource you are trying to access requires you to re-authenticate."
-)
-_DEFAULT_CONSENT_REAUTH_MESSAGE = (
-    "The resource you are trying to access requires you to "
-    "consent to additional access for the Globus CLI."
-)
-
-
 @sdk_error_handler(
-    error_class="GlobusAPIError",
-    condition=lambda err: err.raw_json is None,
+    error_class="GlobusAPIError", condition=lambda err: err.raw_json is None
```

Comment:
Do you have a linter set up that is rewriting these lines? This seems to have simply been a removal of a comma and a subsequent reflow of the lines throughout this commit.

Reply:
Not tool-driven. The reflow you see was all manual -- I was trying to get a handle on the content here during the semantic refactor and touched a lot of the content.

I think the decorator lines are a little regrettable as a change I made at the time. If I could do it over again I'd leave those alone. But the message emission `click.echo` and `click.secho` usages were the parts I was actively rewriting and trying to find my way to a uniform style.

Do you want me to pull these back? (I assume not worth the effort to rewrite the history, but I'm happy to revert this part of the change if that would be nice.)

Reply:
No need to revert! If it was a linter I would just want to make sure that we agree to use it and then do a mass mechanical PR, rather than seeing extra changes coming through in a trickle. :+1: